### PR TITLE
Fixes #7563: Change AGPL to GPL in the packaging

### PR DESCRIPTION
**Pull request by amousset**
http://www.rudder-project.org/redmine/issues/7563

### CODE REVIEW

File: rudder-reports/debian/copyright [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
 
-    <put author's name and email here>
-    <likewise for another author>
+    Normation - http://www.normation.com
 
 Copyright:
 
-    <Copyright (C) YYYY Name OfAuthor>
-    <likewise for another author>
+    Copyright (C) 2011 Normation SAS
```
Comment (amousset) [inline rudder-reports/debian/copyright]: This copyright file was not configured, unsure about what to put here, and if it needs a separate ticket.

File: rudder-reports/debian/copyright [hunk 2]
```diff
@@ -1,24 +1,15 @@
-This package was debianized by root <root@rudderVM.normation.com> on
-Mon, 17 Jan 2011 08:19:10 -0500.
+This package was debianized by:
 
-It was downloaded from <url://example.com>
+    Jonathan Clarke <Jonathan Clarke <jonathan.clarke@normation.com>> on Tue, 06 Oct 2009 01:20:26 +0200
```
Comment (jooooooon) [inline rudder-reports/debian/copyright]: Can you keep the original date please?

File: rudder-reports/debian/copyright (hunk 1 above)
Comment (jooooooon) [inline rudder-reports/debian/copyright]: Ideally, this should read Copyright (C) 2011-2015 Normation SAS

Comment (jooooooon) [inline rudder-reports/debian/copyright]: No need for a separate ticket

File: rudder-reports/debian/copyright (hunk 2 above)
Comment (fanf) [inline rudder-reports/debian/copyright]: Perhaps there is a reason for the will of change. @amousset: why do you changed it?

**Comment by amousset** [comment]
PR updated.

File: rudder-reports/debian/copyright [hunk 3]
```diff
@@ -1,24 +1,15 @@
-This package was debianized by root <root@rudderVM.normation.com> on
-Mon, 17 Jan 2011 08:19:10 -0500.
+This package was debianized by:
 
-It was downloaded from <url://example.com>
+    Jonathan Clarke <Jonathan Clarke <jonathan.clarke@normation.com>> on Mon, 17 Jan 2011 08:19:10 -0500
```
Comment (jooooooon) [inline rudder-reports/debian/copyright]: This is broken (double name and brackets)

File: rudder-reports/debian/copyright (hunk 2 above)
Comment (amousset) [inline rudder-reports/debian/copyright]: @fanf There is no reason, it was a mistake.

File: rudder-webapp/SOURCES/rudder-root-rename [hunk 4]
```diff
@@ -6,31 +6,33 @@ set -e
...
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program. If not, see <http://www.gnu.org/licenses/agpl.html>.
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (fanf) [inline rudder-webapp/SOURCES/rudder-root-rename]: Why not http://www.gnu.org/licenses/gpl-3.0.txt  ?

Comment (amousset) [inline rudder-webapp/SOURCES/rudder-root-rename]: The text comes from http://www.gnu.org/licenses/gpl-howto.html.

File: rudder-reports/debian/copyright (hunk 3 above)
Comment (amousset) [inline rudder-reports/debian/copyright]: Fixed (there, and in the other files).

File: rudder-webapp/SOURCES/rudder-root-rename (hunk 4 above)
Comment (fanf) [inline rudder-webapp/SOURCES/rudder-root-rename]: ok, perfect

**Pull request merged**